lemminx-maven/src/main/java/org/eclipse/lemminx/extensions/maven/searcher/RemoteCentralRepositorySearcher.java

```diff
@@ -240,20 +245,7 @@ public RemoteCentralRepositorySearcher() {
 				cache);
 	}
 
-	private OkHttpAetherClient newHttpClient() {
-		AetherClientConfig config = new AetherClientConfig();
-		config.setConnectionTimeout(ConfigurationProperties.DEFAULT_CONNECT_TIMEOUT);
-		config.setRequestTimeout(ConfigurationProperties.DEFAULT_REQUEST_TIMEOUT);
-		// config.setSslSocketFactory(SSLContext.getDefault().getSocketFactory());
-
-		// Authentification
-		AetherClientAuthentication authentication = null;
-		final String username = System.getProperty("http.proxyUser");
-		final String password = System.getProperty("http.proxyPassword");
-		if (username != null && password != null) {
-			authentication = new AetherClientAuthentication(username, password);
```

Comment:
Is support for passords in proxies entirely lost, or is it still handled without explicit code?

Reply:
I suppose it's completely lost here.
I'm looking into using java http client and its proxy support using system wide setting - if it doesn't support user/password then we can add it there.

Reply:
Not exactly like this. All the examples say a `java.net.http.HttpClient` user should read authentication requisites (username/password/etc) and provide an `Authenticator` implementation that uses them.

Added a simple Password Authenticator implementation, however it's not tested as well as in old version with Takagi OkHttp3